### DISCUSSION TITLE
Fullscreen mode

### DISCUSSION THREAD

**Discussion by g-busch** [tannerhelland/PhotoDemon#488] [Ideas]
I don't  know if this has been suggested before, but it would be nice to be able to view loaded images in fullscreen. I'm not suggesting an actual image viewer with browsing ability; this may have been suggested a long time ago, but it's probably beyond the scope of PhotoDemon, which is already loaded with features. What I find myself doing lately is closing all the side panels and hiding the status bar. This still leaves me with the menu bar., and is rather time consuming. Fullscreen mode would, of course, temporarily hide everything but the image itself. Perhaps it could be accomplished by a single command that would do all that on its own.

**Comment by hi5** [comment]
Jasc Paint Shop pro has a "edit/view full screen" option which is nice from time to time.

@g-busch if you happen to use [AutoHotkey](https://www.autohotkey.com) you could automate it by sending keystrokes to the PD window to turn on/off certain panels. OR if you use "show full path in title bar" (see PD options) you could "grab" the path and launch an image viewer such as imagine or irfanview in full screen mode.

**Reply by g-busch** [reply]
Thank you. My original approach was to use a viewer which could launch PhotoDemon as an external program, such as Faststone Image Viewer, XnView, etc., but I didn't like either of those two programs, and don't need a file browser per se. BTW, I tried Windows 7 Photo Viewer, which is the worst image viewer I've ever used.

I currently use PD as my default (and only) imaging program, and start it with a double-click in Windows Explorer on an image thumbnail or shortcut. If I want to browse, I switch on Explorer's preview panel and go from there.

Thanks for your suggestion about AutoHotKey. I may just keep doing what I'm doing. Besides, PD has everything else that I could ever need.
